docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,274 @@
+In this tutorial, we prepare a PyTorch Fashion-MNIST example which completed by BigDL and supports running on yarn cluster. In particular, we will show you:
+* How to prepare for the environment before submitting the program to Yarn;
+* How to execute the BigDL program on Yarn through multiple ways;
+
+BigDL Orca example code can be found here [Fashion_MNIST Example](https://github.com/intel-analytics/BigDL/tree/main/docs/docs/tutorials/tutorial_example/Fashion_MNIST/).
+
+# Key Concepts
+## OrcaContext
+A BigDL Orca program usually starts with the initialization of OrcaContext. We can specify the `runtime` (spark or ray) and `cluster_mode` arguments to create or get a SparkContext or RayContext with optimized configurations for BigDL performance.
+
+## Orca Estimator
+After initializing the OrcaContext, we could simply create an Estimatornow, and the Estimator will replicate the model on each node in the cluster, feed the data partition on each node to the local model replica, and synchronize the model parameters using various backend technologies.
+
+Let's get started!
+# Prepare Environment
+Before submitting the BigDL program to Yarn Cluster, we need to setup the environment as the following steps:
+* Setup JAVA Environment
+* Setup Spark Environment
+* Setup Hadoop Environment
+* Install All Needed Python Libraries 
+* For CDH Users
+
+## Setup JAVA Environment
+We need to download and install JDK in the environment, and properly set the environment variable JAVA_HOME, which is required by Spark. JDK8 is highly recommended.
+
+```bash
+# For Ubuntu
+sudo apt-get install openjdk-8-jre
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/
+
+# For CentOS
+su -c "yum install java-1.8.0-openjdk"
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.282.b08-1.el7_9.x86_64/jre
+
+export PATH=$PATH:$JAVA_HOME/bin
+java -version  # Verify the version of JDK.
+```
+
+## Setup Spark Environment
+We need to set environment variables `${SPARK_HOME}` as follows:
+```bash
+export SPARK_HOME=the folder path where you extract the Spark package
+```
+
+## Setup Hadoop Environment
+Check the Hadoop setup and configurations of our cluster. Make sure we correctly set the environment variable HADOOP_CONF_DIR, which is needed to initialize Spark on YARN:
+
+```bash
+export HADOOP_CONF_DIR=the directory of the hadoop and yarn configurations
+```
+
+## Install Needed Python Libraries
+We need first to use conda to prepare the Python environment on the local machine where we submit our application. Create a conda environment, install BigDL and all the needed Python libraries in the created conda environment:
```

Comment:
Organize this section as follows:
1) Use conda to install python libraries
2) Install BigDL
3) Install extra Python dependency